### PR TITLE
[@svelteui/core] add Blockquote component

### PR DESCRIPTION
**Pull request by khalibloo**
## Description

Added Blockquote component.

image
image
image

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing guide](https://github.com/svelteuidev/svelteui/blob/main/CONTRIBUTING.md)
- [x] Prefix your PR title with `[@svelteui/core]`, `[@svelteui/actions]`, `[@svelteui/motion]`, `[@svelteui/core]`, `[core]`, or `[docs]`.
- [x] Include a description of the changes made in the PR description and in the commit messages.
- [x] Include screenshots/videos of the changes made.
- [x] Verify that the linter and tests are passing, with `yarn lint` and `yarn test` or just run `yarn prepush`.

### CODE REVIEW

File: packages/svelteui-core/src/components/Blockquote/Blockquote.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { HTMLAttributes } from 'svelte/elements';
+import type { SvelteUIColor, SvelteUINumberSize } from '$lib/styles';
+import { TextProps } from '../Text/Text';
+
+export interface BlockquoteProps extends TextProps<HTMLHeadingElement> {
```
Comment (BeeMargarida) [inline packages/svelteui-core/src/components/Blockquote/Blockquote.d.ts:5]: We should also extend `HTMLBlockquoteAttributes` here, for the `cite` attribute

File: packages/svelteui-demos/src/demos/core/Blockquote/Blockquote.demo.noicon.svelte [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<script lang="ts" context="module">
+	import type { CodeDemoType, CodeDemoConfiguration } from '$lib/types';
+
+	const code = `
+<script>
+  import { Blockquote } from '@svelteuidev/core';
+
+  const text = \`Money is just one thing, folks. There's a lot of other one things that are everything. There's
```
Comment (BeeMargarida) [inline packages/svelteui-demos/src/demos/core/Blockquote/Blockquote.demo.noicon.svelte:8]: oops :eyes: it doesn't match the text below

**Review by BeeMargarida — COMMENTED** [review]
Looking good :ok_hand: just two small comments

File: packages/svelteui-core/src/components/Blockquote/Blockquote.d.ts (hunk 1 above)
Comment (khalibloo) [reply packages/svelteui-core/src/components/Blockquote/Blockquote.d.ts]: The `cite` does not receive any props from the user, so I haven't added it to the types. Should we add `citeProps` and spread it in the `cite`? Though, I can't think of any use case for this. Being a slot, the user can inject any markup they want into it.

**Comment by BeeMargarida** [comment]
Thank you!! :raised_hands:

**Pull request merged**